demo/README.md

```diff
@@ -8,24 +8,49 @@ The following will provide a Tekton pipeline that can build an example python ap
 
 The details of each build type and the steps required to support Pelorus can be found in the [tekton manifest](./tekton-demo-setup/03-build-and-deploy.yaml)
 
-Example:
+
+Instructions:
+
+* Fork the konveyor/pelorus git repository to your github organization.
+
+* Add or configure your github-secret with your Github token
+
 ```
-./demo-tekton.sh -g https://github.com/konveyor/pelorus.git -b binary
+oc create secret generic github-secret --from-literal=TOKEN=ghp_<snip> -n pelorus
+```
+
+* Copy the sample Pelorus instance manifest in pelorus/demo/operator_tekton_demo_values.yaml.sample
+
+```
+cp pelorus/demo/operator_tekton_demo_values.yaml.sample pelorus/demo/operator_tekton_demo_values.yaml
+```
+
+* Edit pelorus/demo/operator_tekton_demo_values.yaml to match your github organization
+
+* Apply the Pelorus instance
+
+```
+oc apply -f pelorus/demo/operator_tekton_demo_values.yaml
+```
+
```

Comment:
before execution, tell user to enable issues and put the `bug` and `production_issue/name=basic-python-tekton` on some issues?

Reply:
yes.. but I'll do it later